docs/docs/reference.md

```diff
@@ -116,17 +116,17 @@
 
 ## Models
 
-::: sbi.utils.get_nn_models.posterior_nn
+::: sbi.neural_nets.factory.posterior_nn
     rendering:
       show_root_heading: true
       show_object_full_path: true
 
-::: sbi.utils.get_nn_models.likelihood_nn
+::: sbi.neural_nets.factory.likelihood_nn
     rendering:
       show_root_heading: true
       show_object_full_path: true
 
-::: sbi.utils.get_nn_models.classifier_nn
+::: sbi.neural_nets.factory.classifier_nn
```

Comment:
how about `inference_factory`? `factory` seems a bit too generic for me, seeing that `embedding_net`s are also built in `neural_nets`.

Reply:
I am happy to change the name of the file to `inference_factory`. Here are the reasons why I went with the more generic name:
1) The `factory.py` lives in the `neural_nets` module, thus, (too me) it is clear that these factory functions are meant/tailored to all nets in this module. Other modules could theoretically have their own `factory.py`.
2) it's not user-face anyway since the user can not directly import from `sbi.neural_nets`.

Do you still want me to change it?